MANIFEST.in

```diff
@@ -11,22 +11,19 @@ include setup.py
 
 include .coveragerc
 
-include plugin-test.sh
-include requirements-docs.txt
-include runtox.py
-
 recursive-include bench *.py
 recursive-include extra *.py
 
 graft testing
 graft doc
+prune doc/en/_build
 
 exclude _pytest/impl
 
 graft _pytest/vendored_packages
 
 recursive-exclude * *.pyc *.pyo
```

Comment:
What about the recursive-excludes added in https://github.com/pytest-dev/pytest/pull/1901/files#diff-97c91a104c431d0c365565d3ac03ac13R24 - should we add them here?